content/app/development/ux/pages/tracks/_index.en.md

```diff
@@ -151,20 +150,20 @@ public override async Task<LayoutSettings> FormatPdf(LayoutSettings layoutSettin
 }
 ```
 
-Som input til metoden får man `layoutSettings` som inneholder default siderekkefølge under propertyen `layoutSettings.Pages.Order`.
-I tillegg får man skjemadataen som er knyttet til steget som skal avsluttes. Denne kan parses til en C# modell som beskrevet lengere oppe på denne siden.
+As input to the method you have `layoutSettings` which contains the default page order under the property `layoutSettings.Pages.Order`.
+In addition, you have the form data that is associated with the step to be completed. This can be parsed to a C# model as described further up on this page.
 
-Ved å manipulere `layoutSettings.Pages.Order` i denne metoden vil man kunne duplisere de sporvalgene som er gjort for sluttbruker.
-MERK! Kallet til PDF handler, vist nedenfor, må ikke fjernes fra `FormatPDF` metoden dersom du har implementert ytterlig logikk for kvitteringen i `PDFHandler.cs`.
+By manipulating `layoutSettings.Pages.Order` in this method, it will be possible to duplicate the dynamic track choices made for the user.
+NOTE! The call to PDF handler, as shown below, must not be removed from the `FormatPDF` method, if you have implemented additional logic for the receipt in `PDFHandler.cs`.
 
 ```cs
 return await _pdfHandler.FormatPdf(layoutSettings, data);
 ```
 
-For å unngå å duplisere logikk vil vi anbefale å lage en metode som manipulerer siderekkefølgen basert på skjemadata og kalle denne både fra `FormatPdf`og `GetPageOrder`.
-Et kodeeksempel på en slik implementasjon følger. Denne kan for eksempel legges i samme klasse som implementerer interfacet IPageOrder for å holde alle logikk for rekkefølge på samme sted.
+To avoid duplicating logic, we recommend creating a method that manipulates the page order based on form data and calling this both from `FormatPDF` and `GetPageOrder`.
```

Comment:
```suggestion
To avoid duplicating logic, we recommend creating a method that manipulates the page order based on form data and calling this both from `FormatPdf` and `GetPageOrder`.
```